dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -0,0 +1,65 @@
+- content_for(:head) do
+  = stylesheet_link_tag    'jquery.handsontable.full', media: 'all'
+  = javascript_include_tag 'jquery.handsontable.full'
+%h1 Search for Teachers
+
+%table
+  %tr
+    %td
+      Email Filter
+    %td
+      %input#emailFilter
+  %tr
+    %td
+      Address Filter
+    %td
+      %input#addressFilter
+
+%br/
+
+%center
+  %a#search
+    %button Search
+
+%br/
+
+#teacherTable
+
+:javascript
```

Comment:
Done (query params rather than URL params and HTML form rather than javascript).

Also converted it to a HAML table. If the need arises, I'll go back to a handsontable.